### ISSUE TITLE
The teacher.pt provided inside model folder, are they pretrained on ImageNet ? 

### ISSUE THREAD

**Issue by sushovanjena** [denguir/student-teacher-anomaly-detection#16]
I want to know, are you pretraining the teacher on ImageNet ?
In the paper, they mentioned that Teacher is pretrained on ImageNet. Is your repo following it ?

**Comment by denguir** [comment]
Hi,
I am not sure that I followed exactly the paper for this, it's a long time ago now.
But what I can tell for sure is that the resnet18 (against which I train the teacher) is pretrained on ImageNet.
See here, you will see the pretrained flag set to True: https://github.com/denguir/student-teacher-anomaly-detection/blob/master/src/AnomalyResnet18.py

**Comment by sushovanjena** [comment]
Yes, that i am clear. As you have provided, ResNet weights for classes separately, I am sure you are training the pretrained ResNet on that Class, ex Carpet. Am i right ?
Just one more important thing is, were you able to reproduce the AUROC results of paper by your method ? Because I am getting lesser AUROC than mentioned in paper.

**Comment by denguir** [comment]
Yes you are right 👍 
Concerning the ROC, the paper is actually not computing the ROC to evaluate their models, instead they compute the PRO (per region overlap). I simply decided to use the ROC for simplicity but implementing the PRO could be a nice exercise !
